sds_data_manager/lambda_code/SDSCode/cognito_signup_message.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
After reading through this, I'm wondering why we need a custom message at all? I think it is generally for formatting, but I don't see much formatting below, so should we just use the default and live with that? I see adding formatting as a follow-up ticket.